portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funfacts =
```

Comment:
Looks like the method is not being called from the html. Where you planning to hook it up later?

Reply:
Yes, it wasn't working then, but I found a typo, and it works now. I'll send another PR for it.